bugtool/cmd/configuration.go

```diff
@@ -229,6 +240,23 @@ func loadConfigFile(path string) (*BugtoolConfiguration, error) {
 	return &c, err
 }
 
+// Listing tc filter/chain/classes requires specific interface names.
+// Commands are generated per-interface.
+func tcInterfaceCommands() ([]string, error) {
+	ifaces, err := net.Interfaces()
+	if err != nil {
+		return nil, fmt.Errorf("could not list network interfaces: %v", err)
```

Comment:
Nit: typically errors are formatted using `%w` when they are being wrapped using `fmt.Errorf()`, see https://stackoverflow.com/a/61287626/2193236 for a decent explanation.

Reply:
Yeah, was wondering about that. Most errors in Bugtool don't wrap their errors so I just went with the flow in this case.

Reply:
@christarazi  I suspect most of this was written before Go 1.13, perhaps it would make more sense to sweep Bugtool and fix all the errors at once?

Reply:
Sure, it's a minimal improvement, but feel free to go for it. Sep. PR makes sense.